Podcast/SearchITunesTableViewController.swift

```diff
@@ -100,6 +101,11 @@ class SearchITunesTableViewController: ViewController, UITableViewDelegate, UITa
         fetchData(query: searchText, completion: nil)
     }
     
+    func searchBarCancelButtonClicked(_ searchBar: UISearchBar) {
+        tableView.isHidden = false
+        tableView.backgroundView?.isHidden = true
```

Comment:
reason for hiding the background view? I guess this is what drew was saying with the two different background views for our table views.